rollup-plugin/spdx-plugin.ts

```diff
@@ -0,0 +1,214 @@
+// SPDX-FileCopyrightText: 2023 spdx contributors
+//
+// SPDX-License-Identifier: MIT
+
+import * as spdx from "../lib/spdx-tools";
+import * as path from "path";
+import { sha1File } from "sha1-file";
+import type { SPDXDocument as SPDX2Document } from "../lib/api/spdx-document";
+
+const SPDX_ID_PREPENDIX = "SPDXRef-";
+const SPDX_DESCRIBES = "DESCRIBES";
```

Comment:
Why does that code need to define constants? Doesn't the tools-ts export them?

Reply:
yes, we could also import those from the library.
Currently, the api is written to take strings for the relationship types, to make it as simple as possible to use. But we could change that and expect the corresponding enum values.